packages/manager/modules/support/src/support.controller.js

```diff
@@ -0,0 +1,46 @@
+import forEach from 'lodash/forEach';
+
+import {
+  EVENTS,
+  STATE_NAME,
+} from './support.constants';
+
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $scope,
+    $transitions,
+  ) {
+    this.$scope = $scope;
+    this.$transitions = $transitions;
+  }
+
+  $onInit() {
+    const transitionCriteria = {
+      from: `${STATE_NAME}.**`,
+    };
+
+    this.unsubscribers = [
+      this.$scope.$on(EVENTS.startLoading, () => this.startLoading()),
+      this.$scope.$on(EVENTS.stopLoading, () => this.stopLoading()),
+      this.$transitions.onStart(transitionCriteria, () => this.startLoading()),
```

Comment:
This should not be needed, you can add @ovh-ux/ng-line-progress to your app and an error page module will be available soon https://github.com/ovh-ux/manager/pull/1209

Reply:
Since this ticket already  received the "quality check ok" label and #1209 isn't yet available, it could be addressed later on.

Thanks for the inputs.